server/plugin/client.go

```diff
@@ -56,9 +57,11 @@ func (c *client) GenerateOAuthToken(encodedFormValues url.Values) (*serializers.
 }
 
 func (c *client) GetUserProfile(id, accessToken string) (*serializers.UserProfile, int, error) {
-	var userProfile *serializers.UserProfile
+	userProfilePath := fmt.Sprintf(constants.PathUserProfile, id)
+	sanitizedPath := path.Clean(userProfilePath)
```

Comment:
if `userProfilePath` does not start with a forward slash, `Clean` e.g. on "../x/y/z" will be "../x/y/z", ie unchanged. A unit test would reveal that.